ui/app/src/components/FolderBrowserTreeView/FolderBrowserTreeView.tsx

```diff
@@ -73,13 +71,7 @@ export function FolderBrowserTreeView(props: FolderBrowserTreeViewProps) {
                     path: withIndexXml in refs.current.tree.childrenByParentPath ? withIndexXml : p
                   })
                 )
-              : checkPathExistence(siteId, withIndexXml).pipe(
-                  map((exists) =>
-                    exists
-                      ? pathNavigatorTreeFetchPathChildren({ id, path: withIndexXml, expand: true })
-                      : pathNavigatorTreeFetchPathChildren({ id, path: p, expand: true })
-                  )
-                );
+              : of(pathNavigatorTreeFetchPathChildren({ id, path: p, expand: true }));
```

Comment:
The FolderBrowserTreeView uses paths without index, so when fetching/expanding using withIndex it was causing an issue when expanding the root path of the dialog.

Reply:
The path existence check was to see if sending withIndex or not, other than that the calls are the same, so that's why I removed it

Reply:
So how does it work now?

Reply:
It used to be:
```
checkPathExistence => if exists fetchPathChildren using withIndex, if not fetchPathChildren with path (without withIndex)
```
now it would fetchPathChildren withouth `withIndex` in both cases, so checkPathExistence is unnecessary

Reply:
But what if it's a folder? Like `articles` in editorial?